requirements.txt

```diff
@@ -1,4 +1,6 @@
 django-compressor<1.6
+django-modelcluster==1.1
+django-taggit==0.18.0
```

Comment:
Where are these new requirements coming from?

Reply:
those packages are pulling the latest version(which breaks) when they come in as dependencies.

Reply:
The Travis config installs `django-modelcluster==3.1` -- do we want `1.1` instead?

Reply:
I fixed it